Testscripts/Windows/VALIDATE-DA.ps1

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+<#
+.Synopsis
+    This script validates Dependency Agent.
+
+.Description
+    The script will install the Latest Dependency Agent and validate install/uninstall
+    scenarios as well as enable/disable scenarios.
+'
+#>
+param([String] $TestParams,
+      [object] $AllVmData)
+function Main {
+    param (
+        [String] $VMName,
+        [String] $Ipv4,
+        [int] $VMPort,
+        [String] $VMUserName,
+        [String] $VMPassword
+    )
+
+    $remoteScriptName = "validate-da"
```

Comment:
This is not the common way we handle the file name only and file extension. Can you change to the other way? It used to the file name in the variable, and we take the file name only and the extension.

Reply:
Changed this to the way I was using it previously